myst.yml

```diff
@@ -0,0 +1,19 @@
+# See docs at: https://mystmd.org/guide/frontmatter
+version: 1
+project:
+  subject: Fornax Demo Notebooks
+  description: Tutorial notebooks for example Fornax use cases
+  keywords: [astronomy]
+  authors: [Fornax developers]
```

Comment:
I don't think this authors field should be displayed on every page. The notebooks each have a list of authors at the bottom. It's probably fine for this to show up on the main page but then it should say "Fornax scientists and developers".